pennylane/ops/channel.py

```diff
@@ -426,13 +504,6 @@ class PauliError(Channel):
 
     num_params = 2
     num_wires = AnyWires
-    par_domain = "L"
```

Comment:
What did L mean here?

Reply:
Lol, no idea, but I must have overlooked it before when removing these things